candidate_models/model_commitments/model_layer_def.py

```diff
@@ -184,7 +184,6 @@ def bagnet():
         ['AvgPool_1a'],
     'mobilenet_v2': ['layer_1'] + [f'layer_{i + 1}/output' for i in range(1, 18)] + ['global_pool'],
     'basenet': ['basenet-layer_v4', 'basenet-layer_pit', 'basenet-layer_ait'],
-    # 'bagnet' : bagnet(),
     'bagnet9':  bagnet(),
     'bagnet17':  bagnet(),
     'bagnet33':  bagnet(),
```

Comment:
this looks incorrect. all identifiers are pointing to the same function without differences in parameters

Reply:
This is just the expansion of the previous hack in the `getItem` method, where `bagnet*` names are resolved to `bagnet` corresponding to the old entry. I did not change the semantics and it also works,

Reply:
Ah sorry you're right, these are just the layers.